src/main/webapp/script.js

```diff
@@ -15,16 +15,27 @@
 const CHART_WIDTH = 800;
 const CHART_HEIGHT = 400;
 
-google.charts.load('current', {'packages': ['corechart']});
-google.setOnLoadCallback(getChart);
+var numCommentsSlider;
 
-/**
- * Fetches comment data from video ID
- */
-async function getYouTubeComments() {
+google.charts.load('current', {'packages':['corechart']});
+google.setOnLoadCallback(getChart)
+
+window.onload = init;
+
+function init(){
+  numCommentsSlider = document.getElementById('num-comments-input');
+  var numCommentsIndicator = document.getElementById('slider-output');
+
+  numCommentsSlider.oninput = function() {
+    numCommentsIndicator.innerText = this.value;
+  }
+}
+
+async function getYouTubeComments() { 
   const urlInput = document.getElementById('link-input');
   const url = cleanseUrl(urlInput.value);
-  const response = await fetch('/YouTubeComments?url='+url);
+  const numComments = numCommentsSlider.value;
```

Comment:
I think it will be clear to just directly use the "num-comments-input" element here instead of having to rely upon a global variable.

I.e. we can replace `numCommentsSlider` with `document.getElementById('num-comments-input')`